src/main/java/prefs/GlobalConfig.java

```diff
@@ -65,28 +66,44 @@ public void clearMarkedReadAt(String repoId, int issue) {
         }
     }
 
-    public void addBoard(String name, List<String> filterExprs) {
-        boards.put(name, filterExprs);
+    public void addBoard(String name, List<PanelInfo> panels) {
+        savedBoards.put(name, panels);
     }
 
-    public List<String> getBoardPanels(String name) {
-        return boards.get(name);
+    public List<PanelInfo> getBoardPanels(String name) {
+        return savedBoards.get(name);
     }
 
-    public Map<String, List<String>> getAllBoards() {
-        return new HashMap<>(boards);
+    public Map<String, List<PanelInfo>> getAllBoards() {
+        return new HashMap<>(savedBoards);
     }
 
     public void removeBoard(String name) {
-        boards.remove(name);
-    }
-
-    public void setLastOpenFilters(List<String> filter) {
-        lastOpenFilters = new ArrayList<>(filter);
+        savedBoards.remove(name);
     }
 
     public List<String> getLastOpenFilters() {
-        return new ArrayList<>(lastOpenFilters);
+        List<String> lastOpenFilters = new ArrayList<>();
+        for (int i = 0; i < openPanels.size(); i++) {
+            lastOpenFilters.add(openPanels.get(i).getPanelFilter());
+        }
+        return lastOpenFilters;
+    }
+    
+    public List<String> getPanelNames() {
+        List<String> openPanelNames = new ArrayList<>();
+        for (int i = 0; i < openPanels.size(); i++) {
+            openPanelNames.add(openPanels.get(i).getPanelName());
+        }
+        return openPanelNames;
+    }
```

Comment:
These two methods seem unnecessary. We already have a way to get the `PanelInfo`. Since they're only used in tests, map over that in the tests instead.